app/assets/javascripts/settings.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Since we'll need to rebase for the merge conflict, could you please remove the two empty files ?
Thanks